treeherder/webapp/api/performance_data.py

```diff
@@ -418,6 +419,19 @@ def _timerange(self, queryset, name, value):
             push__time__gt=datetime.datetime.utcfromtimestamp(int(time.time() - int(value)))
         )
 
+    def _show_sheriffed_frameworks(self, queryset, name, value):
+        return queryset.filter(
+            framework__name__in=[
```

Comment:
Create a new constant and import it
```
    SHERIFFED_FRAMEWORKS = [
        "browsertime",
        "raptor",
        "talos",
        "awsy",
        "build_metrics",
        "js-bench",
        "devtools",
    ]
```
in ui/perfherder/perf-helpers/constants.js